lib/init/grass.py

```diff
@@ -1585,7 +1585,7 @@ def say_hello():
 
             revision = linerev.split(" ")[1]
             sys.stderr.write(" (" + revision + ")")
-        except:
+        except Exception:
```

Comment:
Doesn't this add add another warning, where we don't use anything in that except?

Reply:
As per flake8, there should be no bare 'except' statements E722 - and at the same time it shouldn't be assigned to an unused variable which will trigger F841. I ran flake8 post this update and it didn't highlight anything